### PR TITLE
ase: update QUARTUS_HOME environment variable to QUARTUS_ROOTDIR

### PR DESCRIPTION
**Pull request by sophimao**
Resolves OFS/opae-sdk#2952

### CODE REVIEW

**Review by michael-adler — COMMENTED** [review]
Why not just replace all instances of QUARTUS_HOME with QUARTUS_ROOTDIR and call it a day?

Is there a need for the new override?

**Comment by sophimao** [comment]
> Why not just replace all instances of QUARTUS_HOME with QUARTUS_ROOTDIR and call it a day?
> Is there a need for the new override?

This is done based on [Mike's comment here](https://github.com/OFS/opae-sdk/issues/2952#issuecomment-1562842905), he found that the [Quartus doc](https://www.intel.com/content/www/us/en/docs/programmable/683472/22-1/setting-environment-variables.html) specifies users to set `QUARTUS_ROOTDIR_OVERRIDE` when they "have multiple copies of the Intel® Quartus® Prime software on the same computer and want to determine which version to open by default".

File: ase/Makefile [hunk 1]
```diff
@@ -417,12 +424,12 @@ ifneq ($(SIMULATOR), VCS)
 endif
 # Check gate simulation libraries
 ifeq ($(GLS_SIM), 1)
-  ifndef QUARTUS_HOME
-	@echo "** ERROR **: env(QUARTUS_HOME) has not been set."
-	@echo "** ERROR **: GLS_SIM needs QUARTUS_HOME environment variable to be set."
+  ifeq ($(QUARTUS_HOME), )
```
Comment (michael-adler) [inline ase/Makefile:427]: I don't think you need this complex check. Just confirm that QUARTUS_ROOTDIR is set. If the override is also set that won't be a problem, but QUARTUS_ROOTDIR is the sufficient requirement.

Then you can avoid setting QUARTUS_HOME above.

Comment (michael-adler) [reply ase/Makefile:427]: Yes, you can use QUARTUS_ROOTDIR for those. I haven't seen any significant changes for the way we use them across many Quartus versions. It ought to be safe to ignore even the override.

**Review by michael-adler — APPROVED** [review]
I didn't try running, but trust you have. This looks simpler. Thanks.

**Pull request merged**